### PR TITLE
add an option to select which columns to display

### PR DESCRIPTION
**Pull request by antocuni**

### CODE REVIEW

**Comment by ionelmc** [comment]
Nice, thanks! 

Something is going on with Travis, it did not start a build. Lemme see if I figure it out ...

**Comment by antocuni** [comment]
good, now travis and appveyor have been triggered. Let's wait for the result :)

File: tests/test_benchmark.py [hunk 1]
```diff
@@ -945,3 +945,18 @@ def test_only_benchmarks(testdir):
...
+def test_columns(testdir):
+    test = testdir.makepyfile(SIMPLE_TEST)
+    result = testdir.runpytest('--doctest-modules', '--benchmark-columns=max,iterations,min', test)
+    result.stdout.fnmatch_lines([
+        "*collected 3 items",
+        "test_columns.py ...",
+        "* benchmark: 2 tests *",
+        "Name (time in ?s) * Max * Iterations * Min *",
+        "------*",
+        "test_fast          * 1  *",
```
Comment (ionelmc) [inline tests/test_benchmark.py]: I think the calibration don't guarantee 1 iteration. Maybe just remove this line and following lies - you can only assert on the header part reliably.

Comment (ionelmc) [inline tests/test_benchmark.py]: PS. This is what causes pypy jobs to fail on travis.

Comment (antocuni) [inline tests/test_benchmark.py]: indeed, thank you for the hint. Let's see if it finally passes now :)

**Pull request merged**